api_v3/lib/types/EntryServerNode/base/KalturaEntryServerNodeStatus.php

```diff
@@ -0,0 +1,12 @@
+<?php
+/**
+ * @package api
+ * @subpackage enum
+ */
+class KalturaEntryServerNodeStatus extends KalturaDynamicEnum implements EntryServerNodeStatus{
```

Comment:
Why does this class extends KalturaDynamicEnum do you have any plugins extending this class?
if not extend KalturaEnum.